pdns/recursordist/Makefile.am

```diff
@@ -216,3 +222,14 @@ $(MANPAGES):
 	exit 1
 endif
 endif
+
+if !HAVE_LUA_HPP
+# no lua.hpp on solaris :(
+lua.hpp:
```

Comment:
We have this code in two places now. How about just shipping our own lua.hpp and never relying on the distro?